lessons/en/phoenix/installing.md

```diff
@@ -0,0 +1,86 @@
+%{
+  version: "1.0.0",
+  title: "Installing Phoenix",
+  excerpt: """
+  With some simple steps you can install and create your first phoenix project
+  """
+}
+---
+
+## Getting Started
+
+### Some considerations
+
+Before aaaaaaaaaaaaaaa creating a Phoenix project, you will need to install elixir in your machine. The latest version of phoenix framework needs [Elixir 12 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later) and [Erlang 22 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later).
+
```

Comment:
```suggestion
Before creating a new Phoenix app we need to ensure we have compatible versions of Elixir installed. The latest version of Phoenix framework needs [Elixir 12 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later) and [Erlang 22 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later).
```

In general we prefer to use "we" in places of "you" or "I" so it the learning process feels collaborative and less directed at someone or about someone. Could you work through the lesson updating those references?

Reply:
Yeah, thanks for reviewing the PR @doomspork . I will update the PR next weekend!